src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObjectManaged.cs

```diff
@@ -17,8 +17,6 @@ internal class TdsParserStateObjectManaged : TdsParserStateObject
         internal SNIPacket _sniAsyncAttnPacket = null;                // Packet to use to send Attn
         private readonly Dictionary<SNIPacket, SNIPacket> _pendingWritePackets = new Dictionary<SNIPacket, SNIPacket>(); // Stores write packets that have been sent to SNI, but have not yet finished writing (i.e. we are waiting for SNI's callback)
 
-        private readonly WritePacketCache _writePacketCache = new WritePacketCache(); // Store write packets that are ready to be re-used
```

Comment:
Why remove this? From what I understood from our offline discussion is that we need to make sure that the `WritePacketCache.Add` code path is not being utilized and that causes new write packets to be created everytime.

Reply:
What I meant in the offline discussion was the is no code path that triggers `_writePacketCache.Add()`. The only spot that executes the method in `TdsParserStateObjectNative` is `RemovePacketFromPendingList()`, and there is no implementation for the method in `TdsParserStateObjectManaged`